### ISSUE TITLE
Cannot import azurerm_frontdoor_custom_https_configuration v0.13.7 v2.58.0

### ISSUE THREAD

**Issue by rhollins** [terraform-providers/terraform-provider-azurerm#11698]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform v0.13.7
+ provider registry.terraform.io/-/azurerm v2.58.0
+ provider registry.terraform.io/-/random v3.1.0
+ provider registry.terraform.io/hashicorp/azurerm v2.58.0
+ provider registry.terraform.io/hashicorp/random v3.1.0

### Affected Resource(s)

I'm trying to run following command to import azurerm_frontdoor_custom_https_configuration which is required after v2.58.0 changes as https config has been separated from front doot resource.

`terraform import module.<module_name>.module.front-door.azurerm_frontdoor_custom_https_configuration.custom_https_<endpoint_name> /subscriptions/<subscription_id>/resourceGroups/<rg_name>/providers/Microsoft.Network/frontDoors/<fd_name>/frontendEndpoints/<endpoint_name>`

and here is the error message:

`Error: Cannot parse Azure ID: parse "": empty URL`

I checked that front door resource exists in tf state and in azure and the id of endpoint is correct

here is how the resource looks like in tf code:

```
resource "azurerm_frontdoor_custom_https_configuration" "custom_https_endpointname" {
  frontend_endpoint_id              = azurerm_frontdoor.front-door.frontend_endpoints["some_name"].id
  custom_https_provisioning_enabled = true

    custom_https_configuration {
      certificate_source                         = "AzureKeyVault"
      azure_key_vault_certificate_vault_id       = var.key_vault_id
      azure_key_vault_certificate_secret_name    = var.key_vault_secret_name
      azure_key_vault_certificate_secret_version = var.key_vault_secret_version
    }
}
```

**Comment by bcline760** [comment]
I brought this up with #11623 and #11642 was merged. So the import capabilities should be coming really soon!

**Comment by favoretti** [comment]
@rhollins Can you retry with `2.59` please?

**Comment by rhollins** [comment]
Thanks it works fine !

terraform import module.<...>.module.front-door.azurerm_frontdoor_custom_https_configuration.<front_endpoint_name> /subscriptions/<...>/resourceGroups/<...>/providers/Microsoft.Network/frontDoors/<...>/customHttpsConfiguration/<front_endpoint_name>

**Comment by favoretti** [comment]
Great to know. Thanks!

**Comment by WodansSon** [comment]
Since this was fixed with #11642 in v2.59.0 I going to go ahead and close this issue. Thanks again for opening! 🚀
